deno.json

```diff
@@ -19,8 +20,9 @@
     "ok": "deno fmt --check && deno lint && deno task check:types && deno task test",
     "wasmbuild": "deno run -A jsr:@deno/wasmbuild@0.17.2 --sync --out src/wasm"
   },
-  "fmt": { "exclude": ["dist", "target"] },
-  "lint": { "exclude": ["dist", "target", "testdata"] },
+  "exclude": ["testdata"],
+  "fmt": { "exclude": ["dist", "target", "!testdata"] },
+  "lint": { "exclude": ["dist", "target"] },
```

Comment:
I had to exclude testdata because of the new typescript files I added that import css files. Before there was only an exclusion for lint.